.github/workflows/release-part-2.yml

```diff
@@ -58,8 +70,6 @@ jobs:
           # move the artifacts out of the folders
           mv artifacts/*/* artifacts
           rm -rf artifacts/Wheel*
-          # verify that all 6 wheels are there
-          if [ `find artifacts/* | wc -l` -ne 6 ]; then exit 1; fi
```

Comment:
Without this check, if we change the name of the wheel artifacts, and forget to update it here, then the release will proceed without the necessary wheels. It seems like you can upload files to a release after it's published, so this shouldn't be a critical issue.

Reply:
Okay, that sounds good.

Actually it has me thinking that the more dangerous part is the artifacts for PyPI, since we can't modify that release after it's published. Like what if we changed the artifact names from "Wheel for *". We might not have any wheels downloaded, but we might still call `twine upload artifacts/natcap.invest*` with the source distribution `natcap.invest....tgz` available.